src/collections/news/2022/2022-05-18-meshmark-the-cloud-native-value-measurement/meshmark-the-cloud-native-value-measurement.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "MeshMark: the Cloud Native Value Measurement"
+subtitle: "An open index for measuring performance of cloud native infrastructure in context of the value provided to your business."
+date: 2022-05-18 08:00:00 -0530
+author: Kate Suttons
+thumbnail: 
```

Comment:
Build is failing because of no thumbnail, should we use this logo itself?

![meshmark-dark-full](https://user-images.githubusercontent.com/85789734/168602142-1568b47a-7478-46a0-b1ba-eacb411ccfb8.svg)